Trust and compliance: Add accessibility page (#98)
pretix/pretix-docs: docs/trust/accessibility.md

# Accessibility

> When creating a ticket shop, accessibility of the shop to everyone is not only important for both ethical and business purposes, but also a legal obligation.

Reviewer: ```suggestion
The accessibility of your pretix ticket shop is not only important from ethics and business standpoints, but also a legal obligation.
```

As of 2025, the [European Accessibility Act](https://en.wikipedia.org/wiki/European_Accessibility_Act) and its national implementation laws mandate that all websites that are used to close contracts with consumers need to be in compliance with accessibility standards.

> This therefore applies to all ticket shops selling tickets to consumers within the EU.

Reviewer: ```suggestion
If you operate an online shop selling products or services to consumers within the EU, then this applies to you. 
This includes ticket shops created using pretix. 
```

> The relevant European norm, [EN 301 549](https://en.wikipedia.org/wiki/EN_301_549), refers to the well-known [WCAG 2.1 AA](https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines) standard.

Reviewer: "well-known" kommt potenziell herablassend herüber
```suggestion
The relevant European norm, [EN 301 549](https://en.wikipedia.org/wiki/EN_301_549), refers to the [WCAG 2.1 AA](https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines) standard.
```

Therefore, we develop pretix in a way that makes it easy for you to achieve **WCAG 2.1/2.2 AA** compliance for your ticket shop.

## Your responsibilities

pretix is not a ticket shop, but rather a software to create ticket shops.

> As such, the real-world accessibility of the shop is defined by a combination of our efforts of creating the software and your efforts of configuring it correctly.

Reviewer: ```suggestion
As such, the real-world accessibility of the shop depends on a combination of our efforts of creating the software and your efforts of configuring it for good accessibility.
```

It is your responsibilities to choose configuration parameters for pretix that are in line with accessibility requirements.

> For example, this includes:

Reviewer: Als Leser würde ich mir an dieser Stelle eine erschöpfende Liste an Vorkehrungen wünschen, die ich treffen muss. 
Gibt es außer den Punkten auf der Liste noch etwas, wie ich die Barrierefreiheit versemmeln kann? 
Wenn ja, dann sollten wir das hier erwähnen.

Author: Ist halt sone Haftungsfrage. Ich glaube, dass das für die meisten Kunden alles ist, aber ich will nicht dafür haften, dass wir nicht doch was übersehen haben, wie man sich selbst in den Fuß schießen kann, in dem man Dinge komisch benutzt.

- Use user-configured colors that are [well-distinguishable](https://www.w3.org/TR/WCAG22/#use-of-color) and provide [sufficient contrast](https://www.w3.org/TR/WCAG22/#contrast-minimum) to surrounding colors.

> - Keep all information [presented in images](https://www.w3.org/TR/WCAG22/#images-of-text) (such as the header image or product pictures) accessible in text form on the same page as well.

Reviewer: ```suggestion
- Provide any information that you [present as images](https://www.w3.org/TR/WCAG22/#images-of-text) (such as the header image or product pictures) also as text on the same page. 
You can do this by providing good alt text. 
```

Author: ```suggestion
- Provide any information that you [present as images](https://www.w3.org/TR/WCAG22/#images-of-text) (such as the header image or product pictures) also as text on the same page. 
```
pretix setzt an diesen Stellen automatische Alt-Texte, zB mit dem Veranstaltungs- oder Produktnamen. Der User hat darauf also keinen Einfluss, und soll einfach Text in diesen Bildern unterlassen

- When providing formatted text using [Markdown](../guides/markdown.md), make sure to make semantically correct use of the markup. 

> We recommend against using headlines (``##``) or HTML tags in most cases.

Reviewer: In welchen Fällen genau raten wir davon ab?

Author: Ich rate in *allen* Fällen davon ab. Wenn man sehr genau weiß, was man tut, und die Dokumentenstruktur danach selbst testet, kann man das schon machen, und aus Abwärtskompatibilitätsgründen kann ichs jetzt schlecht verbieten, aber wenn man mich nach einer Empfehlung fragt: Einfach lassen.

Reviewer: Okay, ich finde in dem Fall kann man das einfach als Empfehlung stehen lassen. So wird es nicht strikt verboten 
```suggestion
We recommend not using headlines (``##``) or HTML tags. 
```

- Select payment providers that provide good accessibility support.

- When embedding the [widget](../guides/widget.md), make sure that the CSS styling on the embedding website does not interfere with the accessibility and make sure that the users path to the widget is also accessible.

## Declaration of accessibility

Depending on your jurisdiction, you might need to publish information on the accessibility of your ticket shop.
You can embed this information using the configuration options at :navpath:Your organizer → :fa3-wrench: Settings → General → Accessibility:. 

Here is a Markdown-formatted sample based on the requirements of German law for a non-governmental entity.

> Please note that we are not lawyers and the example it is given **without warranty**.

Reviewer: ```suggestion
!!! Warning 
    The pretix documentation is not legal advice. 
    The information below is provided **without warranty**. 
```

Feel free to use it at your own risk.

```
TODO
```

## Accessibility testing

> To ensure that our ticket shops are accessible, we have conducted testing of by external experts.

Reviewer: Was wurde getestet? Die Software oder die Ticketshops, die wir damit erstellt haben? 
```suggestion
External experts have tested the accessibility of our shops. 
```

Author: Eine Software kann man nicht testen. Alle Ticketshops haben wir auch nicht getestet (das könnte deine Aussage suggerieren). Es wurden Beispielshops getestet. Das wird unten dann ja genauer ausgeführt.

Reviewer: ```suggestion
External experts have tested the accessibility of several example shops. 
```

These last round of tests was between November 2024 and May 2025.

!!! Note

>     At the moment, our accessibility commitment is only valid for the ticket shops created with pretix.

Reviewer: Das "commitment" ist für alles relevant, auch wenn die Tests und Updates noch nicht passiert sind.

Author: Nein, wir haben gegenüber niemandem ein committment auf accessibilty abgegeben fürs Backend, uns zwingt auch kein Gesetzgeber dazu, und selbst wenn wir da Schritt für Schritt besser werden wollen, möchte ich auch hier jetzt kein verbindliches commitment dazu abgeben.

>     For other functionality, such as the pretix backend, we are also working to improve accessibility, but we have prioritized the ticket shops and have not yet performed intensive accessibility testing and improvements on the backend.

Reviewer: ```suggestion
    We are working on the implementation of accessibility standards on the pretix backend, but we have not yet performed intensive accessibility testing or updated the backend accordingly. 
    Our current priority are the ticket shops. 
```

Author: Nein, ich möchte nicht public darauf committen, dass wir uns im backend irgendwannan alle standards halten werden, solange uns niemand dazu zwingt. Manche der Standards werden nämlich total dämlich ausgelegt, und solange kein Gesetz entgegen steht, treffen wir manche Entscheidungen lieber nach "bestem wissen und gewissen" als nach "strengster Auslegung des Wortlaut des Standards"

```suggestion
    We are working on improving accessibility of the pretix backend, but we have not yet performed intensive accessibility testing or updated the backend accordingly. 
    Our current priority are the ticket shops. 
```

### Scope of testing

> Due to the complex and highly configurable nature of pretix, it is impossible to test all different configurations that pretix may be used in.

Reviewer: ```suggestion
Due to the complex and highly configurable nature of pretix, it is impossible to test all possible configurations.
```

Therefore, we performed the test on sample ticket shops and attempted to include as many pretix features and configurations as possible.
Configurations, plugins or processes not listed here have not been included in our accessibility tests.
Should you require a proof of compliance for your specific ticket shop that uses pretix, you will need to perform your own testing.

The accessibility test included core system functionalities as well as the following plugins:

- Bank transfer

> - Double-Opt-In-Step

Author: An der orthografisch komische Benennung des Plugins bin ich unschuldig… aber die inkonsistenz können wir uns erlauben

- Passbook tickets
- PDF ticket output
- Newsletter integration (rapidmail used as an example)
- Pages
- Seating
- Shipping

> The following features of pretix were used as part of the accessibility test:

Reviewer: Würde den Satz hier entfernen und stattdessen die drei folgenden Unterabschnitte mit einem ähnlichen Satz einleiten
```suggestion

```

#### Organizer pages

> 

Reviewer: ```suggestion

The accessibility tests included the following functions on organizer pages:

```

- Switching the page language.
- Exploring the list of events through list and calendar views.
- Getting in touch with the organizer.
- Creating a customer account.
- Logging in to a customer account.
- Performing a customer account password reset.
- Updating customer account information.
- Viewing memberships, recent orders, stored addresses, and profiles.

#### Event pages

> - Navigating the event dates of an event series.

Reviewer: ```suggestion
- Navigating the dates of an event series.
```

- Discovering all event information (date, time, location, description).

> - Discovering and navigating the list of products including reduced tickets, sold-out tickets, tickets with variations, tickets with a waiting list, tickets that require a voucher.

Reviewer: ```suggestion
- Discovering and navigating the list of products including reduced tickets, sold-out tickets, tickets with variations, tickets with a waiting list, and tickets that require a voucher.
```

- Discovering and navigating the seating plan.

> - Signing up for the waiting list of a sold-out ticket.

Reviewer: ```suggestion
- Signing up for the waiting list for a sold-out ticket.
```

> - Adding tickets to the cart and interacting with cart.

Reviewer: ```suggestion
- Adding tickets to the cart. 
- Interacting with the cart. 
```

- Selecting add-on and cross-selling products.
- Going through checkout without creating a customer account, using an existing customer account, or creating a customer account.

> - Answering custom questions of various types.

Author: Hm. Das ist nicht das selbe. "Fragen unterschiedlicher Typen" ist mehr als "unterschiedliche Fragen". Die Typen sind accessibility-relevant, zB weil sich ein Datumsfeld anders verhält als ein Zahlenfeld.

> - Answering custom questions of various types.

Reviewer: Okay, dann passt das.

- Completing the double-opt-in step.
- Selecting a shipping method.

> - Selecting a payment method (without testing of external components of payment methods).

Reviewer: ```suggestion
- Selecting a payment method (We did not test any external components of payment methods).
```

# Accessibility

The accessibility of your pretix ticket shop is not only important from ethics and business standpoints, but also a legal obligation.
As of 2025, the [European Accessibility Act](https://en.wikipedia.org/wiki/European_Accessibility_Act) and its national implementation laws mandate that all websites that are used to close contracts with consumers need to be in compliance with accessibility standards.
If you operate an online shop selling products or services to consumers within the EU, then this applies to you. 
This includes ticket shops created using pretix. 

The relevant European norm, [EN 301 549](https://en.wikipedia.org/wiki/EN_301_549), refers to the [WCAG 2.1 AA](https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines) standard.
Therefore, we develop pretix in a way that makes it easy for you to achieve **WCAG 2.1/2.2 AA** compliance for your ticket shop.

## Your responsibilities

pretix is not a ticket shop, but rather a software to create ticket shops.

> As such, the real-world accessibility of the shop depends on a combination of our efforts of creating the software and your efforts of configuring it for good accessibility.

Reviewer 2: ```suggestion
As such, the real-world accessibility of the shop depends on a combination of our efforts in creating the software and your efforts in configuring it for good accessibility.
```

Über das "efforts of" stolpere ich sehr. Vielleicht reicht die Änderung auf "in" schon; muss ich mir mit mehr Abstand noch mal angucken.

It is your responsibility to choose configuration parameters for pretix that are in line with accessibility requirements.
This includes, but is not limited to:

> 

Reviewer 3: ```suggestion

- Use simple language e.g. for your event description, product titles, etc.

```

Author: habs eingefügt aber am ende

> - Use user-configured colors that are [well-distinguishable](https://www.w3.org/TR/WCAG22/#use-of-color) and provide [sufficient contrast](https://www.w3.org/TR/WCAG22/#contrast-minimum) to surrounding colors.

Author: Die Docs machen ja grundsätzlich sowas gerne als Aufforderung an den User, daher hätte ich die Stichpunkte jetzt eher so gelassen, gerne mit einer anderen Einleitung. Ich glaube aber, so stört es vermutlich auch niemanden wirklich

> - Use user-configured colors that are [well-distinguishable](https://www.w3.org/TR/WCAG22/#use-of-color) and provide [sufficient contrast](https://www.w3.org/TR/WCAG22/#contrast-minimum) to surrounding colors.

Reviewer 2: > These are some steps you should take to fulfill these requirements:

- Provide any information that you [present as images](https://www.w3.org/TR/WCAG22/#images-of-text) (such as the header image or product pictures) also as text on the same page. 

- When providing formatted text using [Markdown](../guides/markdown.md), make sure to make semantically correct use of the markup. 
We recommend not using headlines (``##``) or HTML tags. 

> - Select payment providers that provide good accessibility support.

Author: Nein. Ich finde keinen, der dazu eine klare Aussage macht, auf die man verlinken könnte. Ich habe Vermutungen, wer sich da mehr oder weniger gut schlägt, aber nichts, was ich öffentlich sagen wollen würde.

The relevant European norm, [EN 301 549](https://en.wikipedia.org/wiki/EN_301_549), refers to the well-known [WCAG 2.1 AA](https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines) standard.
Therefore, we develop pretix in a way that makes it easy for you to achieve **WCAG 2.1/2.2 AA** compliance for your ticket shop.

## Your responsibilities

pretix is not a ticket shop, but rather a software to create ticket shops.
As such, the real-world accessibility of the shop is defined by a combination of our efforts of creating the software and your efforts of configuring it correctly.
It is your responsibilities to choose configuration parameters for pretix that are in line with accessibility requirements.
For example, this includes:

- Use user-configured colors that are [well-distinguishable](https://www.w3.org/TR/WCAG22/#use-of-color) and provide [sufficient contrast](https://www.w3.org/TR/WCAG22/#contrast-minimum) to surrounding colors.

- Keep all information [presented in images](https://www.w3.org/TR/WCAG22/#images-of-text) (such as the header image or product pictures) accessible in text form on the same page as well.

> - When providing formatted text using [Markdown](/guides/markdown.md), make sure to make semantically correct use of the markup. We recommend against using headlines (``##``) or tables in most cases.

Reviewer: Dateilink und Zeilenumbruch
```suggestion
- When providing formatted text using [Markdown](../guides/markdown.md), make sure to make semantically correct use of the markup. 
We recommend against using headlines (``##``) or tables in most cases.
```
